scripts/deploy.sh

```
@@ -0,0 +1,438 @@
+
+# Disable and remove plugin directory before upload to avoid "destination already exists" errors
+echo "Disabling plugin before upload..."
+curl -s -o /dev/null \
+  -X POST \
+  -H "Authorization: Bearer $MM_TOKEN" \
+  "$MM_URL/api/v4/plugins/$PLUGIN_ID/disable"
+curl -s -o /dev/null \
+  -X DELETE \
+  -H "Authorization: Bearer $MM_TOKEN" \
+  "$MM_URL/api/v4/plugins/$PLUGIN_ID" || true
+rm -rf "$_ABS_PLUGIN_DIR/$PLUGIN_ID" 2>/dev/null || true
```

Shellcheck (0.11.0)

[warning] Use "${var:?}" to ensure this never expands to / .

(SC2115)